source/main/index.ts

```diff
@@ -134,91 +138,89 @@ const onAppReady = async () => {
     ram,
     startTime,
   });
-
   // We need DAEDALUS_INSTALL_DIRECTORY in PATH in order for the
   // cardano-launcher to find cardano-wallet and cardano-node executables
   process.env.PATH = [
     process.env.PATH,
     process.env.DAEDALUS_INSTALL_DIRECTORY,
   ].join(path.delimiter);
-
-  logger.info(`Daedalus is starting at ${startTime}`, { startTime });
-
+  logger.info(`Daedalus is starting at ${startTime}`, {
+    startTime,
+  });
   logger.info('Updating System-info.json file', { ...systemInfo.data });
-
   logger.info(`Current working directory is: ${process.cwd()}`, {
     cwd: process.cwd(),
   });
-
-  logger.info('System and user locale', { systemLocale, userLocale });
-
+  logger.info('System and user locale', {
+    systemLocale,
+    userLocale,
+  });
   ensureXDGDataIsSet();
   await installChromeExtensions(isDev);
-
+  // @ts-ignore ts-migrate(2554) FIXME: Expected 2 arguments, but got 1.
   logger.info('Setting up Main Window...');
```

Comment:
The purpose of this branch, is to migrate, without modifying code unless absolutely necessary, so that we can assume the application will function the same. I would like to create a number of further PRs to address tech debt of adding ts-ignore statements